src/migrations/1692624998160-category-seed.ts

```diff
@@ -0,0 +1,102 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class Migrations1692624998160 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    const update = async (name: string, newName: string) => {
+      await queryRunner.query(`
+            UPDATE category
+            SET name='${newName}'
+            WHERE name='${name}'
+        `);
+    };
+
+    // update categories for language
+    await update('English', 'english');
+    await update('French', 'french');
+    await update('German', 'german');
+
+    // add new languages
+    await queryRunner.query(`INSERT INTO category (name, type) VALUES (
+        'arabic',
+        'language'
+        );
+    `);
+    await queryRunner.query(`INSERT INTO category (name, type) VALUES (
+        'italian',
+        'language'
+        );
+    `);
+
+    // update categories for discipline
+    await update('Arts', 'arts');
+    await update('Language', 'language');
+    await update('Math', 'mathematics');
+    await update('Literature', 'literature');
+    await update('Natural Science', 'natural-science');
+    await update('Social Science', 'social-science');
+
+    // update categories for level
+    await update('Kindergarden', 'kindergarten');
+    await update('Primary School', 'primary-school');
+    await update('Lower Secondary School', 'lower-secondary-education');
+    await update('Upper Secondary School', 'upper-secondary-education');
+    await update('Higher Education', 'higher-education');
+    await update('Vocational Training', 'vocation-training');
+
+    // add resource type categories
+    await queryRunner.query(`INSERT INTO category (name, type) VALUES (
+      'app',
+      'resource-type'
+      );
+    `);
+    await queryRunner.query(`INSERT INTO category (name, type) VALUES (
+      'collection',
+      'resource-type'
+      );
+    `);
+    await queryRunner.query(`INSERT INTO category (name, type) VALUES (
+      'template',
+      'resource-type'
+      );
+    `);
```

Comment:
Question: We will have to implement something in the frontend to set these right ?

Reply:
At least add it in the enum. Then I think Builder will automatically update, but not sure. I don't know for library.